tests/cases/fourslash/server/jsdocCallbackTag.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path="../fourslash.ts"/>
+
+// @allowNonTsExtensions: true
+// @Filename: jsdocCallbackTag.js
+
+//// /**
+////  * @callback FooHandler
+////  * @param {string} eventName

```

Comment:
Note that the parser in services almost certainly won't skip it or retrieve it correctly. I don't think that's a problem.